fugue/execution/native_execution_engine.py

```diff
@@ -7,6 +7,7 @@
     EMPTY_PARTITION_SPEC,
     PartitionCursor,
     PartitionSpec,
+    _parse_presort_exp,
```

Comment:
I need this in order to parse the limit presort. I know it's bad practice to import these _functions. Should we change it?